src/test/java/org/jenkinsci/plugins/stashNotifier/DefaultApacheHttpNotifierTest.java

```diff
@@ -154,4 +166,18 @@ public void notifyStash_fail() throws Exception {
         NotificationResult notificationResult = notifyStash(400);
         assertThat(notificationResult.indicatesSuccess, is(false));
     }
+
+    @Test
+    public void notifyStashUsesRequestParameters() throws Exception {
```

Comment:
Thanks! I was hoping to get a test that demonstrates the server side cookie warning goes away with this setting, but I recognize that may be quite difficult to put together.

This approach at least clarifies it's intentional to set this configuration.